draft-ietf-rtcweb-jsep.xml

```diff
@@ -2315,7 +2315,42 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         bounds, or cannot be applied, processing MUST stop and an error MUST be
         returned.
           <list style="symbols">
-            <t>TODO</t>
+            <t>Basic sanity checking is applied for the following attributes in the media section:
+              <list style="symbols">
+                <t>ICE lite: MUST NOT be present.</t>
+                <t>ICE ufrag: MUST be present, and MUST meet the length criteria from [TODO: ref]</t>
+                <t>ICE pwd: MUST be present, and MUST meet the length criteria from [TODO: ref]</t>
+                <t>DTLS fingerprint: MUST be present, and, for existing media sections, MUST be unchanged.</t>
+                <t>DTLS setup: for new media sections, MUST follow the guidance from [TODO: ref]; for existing media sections, MUST be unchanged. [TODO: check whether this is what we agreed on (vs actpass)]</t>
+              </list>
+            </t>
+            <t>If this media section is new, begin gathering candidates for it,
+             as defined in <xref target="RFC5245"/>, Section X.Y.,
+             unless it has been marked as bundle-only.</t>
+            <t>Or, if the ICE ufrag and password values have changed, trigger the ICE Agent
+              to start an ICE restart and begin gathering new candidates for the
+              media section, as defined in <xref target="RFC5245"/>, Section X.Y.,
+              unless it has been marked as bundle-only.</t>
+            <t>If the media section proto value indicates use of RTP:
```

Comment:
This conditional seems weird, because you could have the ufrag/password change *and* have a session-level ICE ufrag/password change. So maybe the opposite order?